corehq/apps/es/case_search_bha.py

```diff
@@ -1,3 +1,22 @@
+"""
+This code supports a clone of the case search index for a single project.
+
+To support more performant case search queries, we write cases to both the main
+case_search index and to this sub-index, when enabled for the case's domain.
+Both indices should have identical sets of cases on that domain, as well as
+identical mappings, so they can be used interchangeably.
+
+Writes are controlled by settings.CASE_SEARCH_SUB_INDICES, which tells us which
+domains to write to this index. This doesn't affect reads, so we can begin
+writing to an index and do the backfill without affecting live usage.
+
+Code reads from the new index when directed there by use of the CaseSearchBhaES
+class, or by passing the proper index cname to CaseSearchES constructor. Case
+search code uses a shared base queryset (QueryHelper.get_base_queryset) which
+checks CaseSearchConfig.index_name to see whether to read from a dedicated
+index. This is edited in the Django admin, which allows us to easily switch app
+users to the dedicated index or back without a deploy.
```

Comment:
```suggestion
index. This is edited in the Django admin, which allows us to easily switch domains
to the dedicated index or back without a deploy.
```
nit: app users makes it sound like the config operates on a per user basis.

Reply:
thanks, updated